src/legacyCompat.ts

```diff
@@ -89,6 +89,21 @@ export async function legacyGetMapFromParams(
 
   const updatedGetMapParams = { ...getMapParams, ...overrideGetMapParams };
 
+  let combinedEffects = {};
+  if (getMapParams.effects && Object.keys(getMapParams.effects).length > 0) {
+    combinedEffects = { ...combinedEffects, ...getMapParams.effects };
```

Comment:
Very minor, but I think there is no need for `...combinedEffects` here.

Reply:
I know, I added it deliberately ... something between paranoia (something not working again because of some stupid mistake) and "just in case I will need to change something in the near future".